### PR TITLE
Added threads.net

### PR DESCRIPTION
**Pull request by deanpcmad**
Threads is a new service by Meta

### CODE REVIEW

**Review by maxxcrawford — APPROVED** [review]
Works as expected

**Comment by ragnvaldr** [comment]
Is www.threads.net needed as well? Current version of the plugin (2.3.9) is not working for me, and threads.net auto-redirects to www.threads.net

**Comment by deanpcmad** [comment]
This hasn't been released yet, hopefully there will be a new release soon

File: src/background.js [hunk 1]
```diff
@@ -35,7 +35,9 @@ const FACEBOOK_DOMAINS = [
 
   "metacareers.com", "meta.com",  "metaque.st",
 
-  "novi.com"
+  "novi.com",
+
+  "threads.net
```
Comment (kherge) [inline src/background.js:40]: It looks like a quote is missing?

**Pull request merged**